ultralytics/utils/metrics.py

```diff
@@ -1190,10 +1191,17 @@ def __init__(self) -> None:
         self.speed = {"preprocess": 0.0, "inference": 0.0, "loss": 0.0, "postprocess": 0.0}
         self.task = "classify"
 
-    def process(self, targets, pred):
+    def process(self, targets, pred, img_names):
```

Comment:
Is this changing all modes? Even detection?
img_names needed everywhere?

Reply:
No, this change is only specific to classification because this "process" method is called from the ClassificationValidator's "get_stats" method, and all of the arguments to this "process" method are specific to the ClassificationValidator.